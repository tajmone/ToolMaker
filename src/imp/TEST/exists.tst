%%IF(foo(?))
Set to <%%(foo)>
%%ELSE
Not set!
%%ENDIF
%%SET foo(1)
%%IF(foo(?))
Set to <%%(foo)>
%%ELSE
Not set!
%%ENDIF
