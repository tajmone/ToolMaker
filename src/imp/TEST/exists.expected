########## exists ##########
Not set!
Set to <1>
